engine/apps/schedules/models/shift_swap_request.py

```diff
@@ -171,10 +171,6 @@ def slack_channel_id(self) -> str | None:
         """
         return self.schedule.channel
 
-    @property
-    def schedule_slack_url(self) -> str:
-        return f"<{self.schedule.web_detail_page_link}|{self.schedule.name}>"
```

Comment:
this has been replaced w/ `slack_url` [here](https://github.com/grafana/oncall/pull/5137/files#diff-4037b7bbef9fc16d9b541beb3ed46f760916d7cd720847c3123adf7afb5ab4b4R260-R262) (and any references to this updated)